custom-plugin-rules/default-enum-match/default-enum-plugin.js

```diff
@@ -0,0 +1,12 @@
+const DefaultEnumMatch = require('./default-enum-match');
+
+module.exports = function myRulesPlugin() {
+  return {
+    id: 'openapi-default-enum',
+    rules: {
+      oas3: {
+        'default-enum-match': DefaultEnumMatch,
```

Comment:
The logic is small, I wonder why separate it to a separate `default-enum-match.js` file? It feels like it could be more difficult for a newer user to adopt if they have to copy multiple files?

Reply:
I see we have very few custom plugin rules. We have more custom plugin decorators. We try to write them into one file when it makes sense, especially when there is a little logic:

https://github.com/Redocly/redocly-cli-cookbook/blob/main/custom-plugin-decorators/azure-apim/azure-apim.js

Reply:
Done